libraries/botbuilder-dialogs/src/prompts/prompt.ts

```diff
@@ -244,6 +263,17 @@ export abstract class Prompt<T> extends Dialog {
         }
     }
 
+    /**
+     * @protected
```

Comment:
This is using `@protected` should we consider adding this to the rest of the protected methods?

Reply:
When searching if it's used in other places I only found one in the whole project, I think its better to not include it. Thanks